changelog/changes/2022-10-07-kernel-enable-bridge-conntrack.md

```diff
@@ -0,0 +1 @@
+- Added CONFIG_NF_CONNTRACK_BRIDGE (for nf_conntrack_bridge) and CONFIG_NFT_BRIDGE_META (for nft_meta_bridge) to the kernel config
```

Comment:
```suggestion
- Added `CONFIG_NF_CONNTRACK_BRIDGE` (for nf_conntrack_bridge) and `CONFIG_NFT_BRIDGE_META` (for nft_meta_bridge) to the kernel config
```
Thanks, since this is for users to read more than developers, it would be good to also summarize the effect/benefit of those

Reply:
(like in the git commit message body)

Reply:
Updated the suggestion